packages/downloading-helpers/src/scripts/replay-assets.ts

```diff
@@ -41,11 +41,11 @@ export const fetchAsset: (path: string) => Promise<string> = async (path) => {
     );
     const filePath = join(await getOrCreateAssetsDir(), assetFileName);
 
-    if (entry && etag !== "" && etag === entry.etag) {
-      logger.debug(`${fetchUrl} already present`);
-      releaseLock();
-      return filePath;
-    }
+    // if (entry && etag !== "" && etag === entry.etag) {
```

Comment:
Is this intentional? Can you add a comment if so?

Reply:
that that not intended, dropped in https://github.com/alwaysmeticulous/meticulous-sdk/pull/473/commits/abc34d9e2253a6c781e3bc84b2873f366dbe08a2